src/Pages/ProductList/ProductList.component.js

```diff
@@ -3,15 +3,87 @@ import React, { Component } from "react";
 import "./ProductList.styles.scss";
 import ProductCard from "./ProductCard/ProductCard.component";
 
+const API = "http://localhost:3000/data/productlist/productlist.json";
+// const API = "http://10.58.4.20:8000/category?main=1"; //Backend API
+
 class ProductList extends Component {
+  state = {
+    isCategoryClick: false,
+    isSortingClick: false,
+    categories: "",
+    products: "",
+  };
+
+  handleSorting = () => {
+    const { isSortingClick } = this.state;
+    this.setState({
+      isSortingClick: !isSortingClick,
+    });
+  };
+
+  componentDidMount = async () => {
+    const response = await fetch(API);
+    // custom error handler
+    if (response.status !== 200) {
+      throw new Error("cannot fetch the data");
+    }
+    const data = await response.json();
+    this.setState({ categories: data.categories, products: data.products });
+  };
+
   render() {
+    const { isSortingClick, categories, products } = this.state;
     return (
       <section className="ProductList">
-        <div className="row">
-          <div className="col s12 m4">
-            <ProductCard />
+        <div className="category-nav">
+          <div className="category-title">
+            <i className="far fa-carrot"></i>
+            <span>채소</span>
+          </div>
+          <div className="category-list">
+            <ul>
+              {categories.length &&
+                categories.map((category) => (
+                  <li key={category.id}>{category.name}</li>
+                ))}
+            </ul>
+            <div
+              className={
+                isSortingClick ? "sort-trigger active-color" : "sort-trigger"
+              }
+              onClick={this.handleSorting}
+            >
+              <span>추천순</span>
+              <i
+                className={
+                  isSortingClick ? "fas fa-chevron-up" : "fas fa-chevron-down"
+                }
+              ></i>
+            </div>
+            <div
+              className={
+                isSortingClick ? "sort-list" : "sort-list display-none"
+              }
```

Comment:
```suggestion
              className={`sort-list ${isSortingClick ? "display-none" : ""}`}
```
템플릿 리터럴을 활용할 수도 있을 것 같긴한데 이 경우는 원래 작성하신 대로 두셔도 무방할 것 같습니다. 참고해주세요.

Reply:
```jsx
className={`sort-trigger ${isSortingClick && "active-color"} `}
```
조기서 약간 변형해서 써보니깐 잘먹히네요! 
한 글자 줄은 느낌이라 기부니 조아요 🤟🏻🤟🏻🤟🏻🤟🏻🤟🏻

근데 어떤 방법이 더 좋은 방법인지 얘기주신대로 살짝 아리까리 하네유 👀👀👀